Maddori.Apple/Maddori.Apple/Screen/View/HomeViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HomeViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/18.
+//
+
+import UIKit
+
+import SnapKit
+
+class HomeViewController: UIViewController {
+    
+    // MARK: - property
+    
+    private let firstLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.keywordType = .defaultKeyword
+        keyword.text = "첫 djdkdkdk번째"
+        return keyword
+    }()
+    private let secondLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.text = "첫 번째"
+        keyword.keywordType = .previewKeyword
+        return keyword
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        render()
+    }
+    
+    // MARK: - func
+    
+    private func configUI() {
+        view.backgroundColor = .black
+    }
+    
+    private func render() {
+        view.addSubview(firstLabel)
+        firstLabel.snp.makeConstraints {
+            $0.left.equalTo(view.safeAreaLayoutGuide).inset(24)
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
+        }
+        view.addSubview(secondLabel)
+        secondLabel.snp.makeConstraints {
+            $0.left.equalTo(firstLabel.snp.right).inset(-10)
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
```

Comment:
p2;

안쪽으로 들어가는 경우라면 inset을 사용하고, 옆으로 떨어지는 방식이라면 offset을 사용할 수 있습니다. 상황에 따라 마이너스를 사용해야 하는 경우도 있긴 하지만 현재로썬 offset 으로 해도 될 것 같네요 ^_______^

```suggestion
            $0.leading.equalTo(firstLabel.snp.trailing).offset(10)
            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
```

Reply:
사실 inset 만 있는 줄 알았습니다 ㅎㅎ
근데 왜 inset의 반댓말이 outset이 아니라 offset일까요...
[d5e69be](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/18/commits/d5e69bec4e17b49194c625a3eb7991602461980c)

Reply:
그건 저도 잘.... 
![image](https://user-images.githubusercontent.com/78677571/196643707-3503b3c7-2fc1-45a9-a76e-e4d103857f2f.png)